tmt/schemas/results.yaml

```
@@ -0,0 +1,58 @@
+---
+
+#
+# JSON Schema definition for tmt `results.yaml` file
+#
+# https://tmt.readthedocs.io/en/stable/spec/plans.html#execute
+#
+
+$id: /schemas/results
+$schema: https://json-schema.org/draft-07/schema
+
+type: array
+items:
+  type: object
+  additionalProperties: false
+
+  properties:
+    guest:
+      type: object
+      additionalProperties: false
+
+      properties:
+        name:
+          type: string
+
+        role:
+          $ref: "/schemas/common#/definitions/role"
+
+      required: []
+      minProperties: 1
+
+    name:
+      type: string
+
```

```suggestion
      pattern: "^/"

```

Should we add pattern specification for the `name` here?

---

Fixed in c2458f14.